adopters.md

```diff
@@ -0,0 +1,107 @@
+D: Equinix uses Flatcar as the OS for its bare metal cloud control plane, which runs in Kubernetes
+      
+    * N: Finleap Connect
+      D: Finleap Connect - At finleap connect we serve over a million financial transactions per day. As a regulated company using a pure cloud-native stack based on Kubernetes, using Flatcar as our foundational building block for reliable, secure and immutable nodes across the public clouds and on bare-metal deployments was a day one decision we never regretted. Today Flatcar serves all of our 12 production clusters with over 300 nodes on public clouds in 3 countries and on our bare-metal private-cloud setup.
+    
+    * N: Host with quantum 
+      C: https://twitter.com/klimpong
+    
+    * N: Intersys AG 
+
+    * N: Spinoco
+
+    * N: Skilld.cloud
+      L: https://www.skilld.cloud
+      D: Flatcar choice was a no-brainer for Skilld: Flatcar is a perfect fit for running Kubernetes workloads. On premise as well as on public clouds. We rely on Flatcar to power up our cutting-edge NRT data-driven ops platforms. A key asset for building distributed & asset management based businesses such as our Community-as-a-service IT platform, or our customers Train fleet's or smart grid's ones.
+
+    * N: Memzo 
+      D: Kinvolk was a valuable source of knowledge when troubleshooting installation issues with our platform vendor. They were able to join us and the vendor on a call and sort out the issues quickly.The use of the Flatcar Update Server gave us confidence about what software/OS versions were running in each of our environments. This allowed us to better test upgrades before promoting the change to production environments.
+
+    * N: Wipro
+      D: Wipro Business Solutions uses Flatcar Linux to power their hybrid/multi-cloud PostgreSQL containerized DBaaS platform. Each provisioned database is running on a dedicated lightweight stack with Flatcar Linux as the foundational OS running on each database VM. In addition the DBaaS API itself and all supporting machines use Flatcar Linux as well. Flatcar Linux has proven to be a well-supported rocket solid OS with minimal attack surface, build in update mechanism and integrated docker daemon. Ignition brings in an early boot provisioning utility that perfectly adds to the full automation approach of the PostgreSQL DBaaS platform. We use it at scale on-prem with OpenStack cloud but also with public clouds like Google and Tencent
+      
+Public anouncmentce: 
+
+    * N: AT&T (DirectTV) 
+      L: https://medium.com/cloud-native-the-gathering/certified-kubernetes-administrator-join-our-team-its-a-good-thing-7e27ab34dc88
+D: "We are integrating Flatcar Container Linux, Istio, OPA, Multi-Region, KNative, and so many other technologies and concepts it makes the mind hurt a bit."
+
+    * N: Darren Shepherd 
+      L:|https://twitter.com/ibuildthecloud/status/1288498089574055938 
+      C: https://twitter.com/ibuildthecloud 
+
+
+    * N: Digital Science 
```

Comment:
N: Digital Science
L: https://digital-science.com
D: We're running Flatcar on all our self-hosted Kubernetes clusters on AWS, used for all data processing behind Dimensions (https://dimensions.ai). We choose Flatcar for security and simplicity.